File: KCD/HMMasterMapAreaCommand.m
//
//  HMMasterMapAreaCommand.m
//  KCD
//
//  Created by Hori,Masaki on 2014/02/16.
//  Copyright (c) 2014年 Hori,Masaki. All rights reserved.
//

#import "HMMasterMapAreaCommand.h"

@implementation HMMasterMapAreaCommand
- (NSString *)dataKey
{
	return @"api_data.api_mst_maparea";
}
- (void)execute
{
	[self commitJSONToEntityNamed:@"MasterMapArea"];
}
@end

File: KCD/HMMasterMapAreaCommand.swift
//
//  HMMasterMapAreaCommand.swift
//  KCD
//
//  Created by Hori,Masaki on 2015/01/11.
//  Copyright (c) 2015年 Hori,Masaki. All rights reserved.
//

import Cocoa

class HMMasterMapAreaCommand: HMJSONCommand {
	override var dataKey: String {
		return "api_data.api_mst_maparea"
	}
	override func execute() {
		super.commitJSONToEntityNamed("MasterMapArea")
	}
}
